static/bidder-info/smartadserver.yaml

```diff
@@ -6,10 +6,12 @@ capabilities:
     mediaTypes:
       - banner
       - video
+      - native
```

Comment:
Could you confirm that this new support for Native is being updated/represented in your docs?

Reply:
Hey @AlexBVolcy, thanks for the heads-up.
We need to update https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/smartadserver.md, right?

However, only the Prebid Server adapter is compatible with Native. The integration with prebid.js directly only supports display/video. How should we proceed in that case?

Thanks!

Reply:
Right, this is a file to update. You may specify native is supported in PBS only, similar to how it's specified here: https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/appnexus.md - Bid params - placement_id